lib/capybara/playwright/browser.rb

```diff
@@ -65,7 +65,7 @@ def visit(path)
             path
           end
 
-          @playwright_page.capybara_current_frame.goto(url)
+          @playwright_page.capybara_current_frame.goto(url, timeout: Capybara.default_max_wait_time * 1000)
```

Comment:
`Capybara.default_max_wait_time` is 2 by default.
It is too short for navigation timeout and this change would break existing tests for many users.